lib/manageiq/appliance_console/cli.rb

```diff
@@ -280,6 +286,26 @@ def set_time_zone
       end
     end
 
+    def set_date_time
+      date_time_config = ManageIQ::ApplianceConsole::DateTimeConfiguration.new
+      unless options[:datetime] == "auto"
+        date_time_config.manual_time_sync = true
+        date_time_config.new_date, date_time_config.new_time = options[:datetime].split("T")
+        return unless date_time_valid?(date_time_config)
+      end
+      date_time_config.activate
+    end
+
+    def date_time_valid?(date_time_config)
```

Comment:
Shouldn't this be `ManageIQ::ApplianceConsole::DateTimeConfiguration#valid?`?

Reply:
I think maybe not. `ManageIQ::ApplianceConsole::DateTimeConfiguration` use `ask_for_date/time` and provide the regexp to `highline`, let `highline` validate if date time is valid. So the `DateTimeConfiguration` doens't validate date and time itself. And these validation only needed in `cli`.